js/piwik.js

```diff
@@ -2516,6 +2582,7 @@ if (typeof Piwik !== 'object') {
              * Send request
              */
             function sendRequest(request, delay, callback) {
+                lastTrackerRequestTime = (new Date()).getTime();
```

Comment:
should we set this variable of lastTrackerRequestTime maybe in the callback of `makeSureThereIsAGapAfterFirstTrackingRequestToPreventMultipleVisitorCreation`? We might not actually send the request there directly. Not sure if i would change anything.